bluesky_queueserver/manager/tests/test_plan_queue_ops.py

```diff
@@ -0,0 +1,585 @@
+import asyncio
+import pytest
+import json
+import copy
+from bluesky_queueserver.manager.plan_queue_ops import PlanQueueOperations
+
+
+errmsg_wrong_plan_type = "Parameter 'plan' should be a dictionary"
+
+
+@pytest.fixture
+def pq():
+    pq = PlanQueueOperations()
+    asyncio.run(pq.start())
+    # Clear any pool entries
+    asyncio.run(pq.delete_pool_entries())
+    yield pq
+    # Don't leave any test entries in the pool
+    asyncio.run(pq.delete_pool_entries())
+
+
+def test_running_plan_info(pq):
+    """
+    Basic test for the following methods:
+    `PlanQueueOperations.is_plan_running()`
+    `PlanQueueOperations.get_running_plan_info()`
+    `PlanQueueOperations.delete_pool_entries()`
+    """
+
+    async def testing():
+
+        assert await pq.get_running_plan_info() == {}
+        assert await pq.is_plan_running() is False
+
+        some_plan = {"some_key": "some_value"}
+        await pq._set_running_plan_info(some_plan)
+        assert await pq.get_running_plan_info() == some_plan
+
+        assert await pq.is_plan_running() is True
+
+        await pq._clear_running_plan_info()
+        assert await pq.get_running_plan_info() == {}
+        assert await pq.is_plan_running() is False
+
+        await pq._set_running_plan_info(some_plan)
+        await pq.delete_pool_entries()
+        assert await pq.get_running_plan_info() == {}
+        assert await pq.is_plan_running() is False
+
+    asyncio.run(testing())
+
+
+# fmt: off
```

Comment:
Is it for `black`?

Reply:
Yes. I think that `@pytest.mark.parametrized` dictionaries should be protected from autoformatting, because they grow large. Also they are manually formatted for readability and convenience of editing. Actually this was one of the issues mentioned by @danielballan when we discussed whether `black` should be used for our projects.

Reply:
Sounds good, I like this workaround.